.lintrunner.toml

```diff
@@ -242,6 +242,7 @@ code = 'CLANGTIDY'
 include_patterns = [
     'aten/src/ATen/core/*.cpp',
     'c10/**/*.cpp',
+    'c10/core/**/*.h',
```

Comment:
```suggestion
    'c10/core/**/*.h*',
```
in case someone ever puts hpp files in there.

Reply:
No hpp in PyTorch

Reply:
There may be someone using hpp in the future. I will change the pattern in the next patch